dist/angular-google-analytics.js

```diff
@@ -590,7 +613,19 @@
               }
 
               if (trackRoutes && !ignoreFirstPageLoad) {
-                _ga(generateCommandName('send', trackerObj), 'pageview', trackPrefix + getUrl());
+                if (readFromState) {
+                  // Evaluate whether send or not the first pageview
+                  // I didn't find a better approach than $timeout since in this context the $state hasn't been loaded yet
+                  $timeout(function() {
```

Comment:
Instead of the timeout is there an event you could listen to? Is the $stateChangeSuccess event sufficient for tracking the first route setup by ui-router? Could this first call simply not happen for ui-router as it will be caught by the event?